lib/webgl.js

```diff
@@ -37,10 +37,25 @@ function createCanvas (element, onDone, pixelRatio) {
     })
   }
 
-  window.addEventListener('resize', resize, false)
+  var resizeObserver
+  if (element !== document.body && typeof ResizeObserver === 'function') {
+    // ignore 'ResizeObserver' is not defined
+    /* eslint-disable */
```

Comment:
Does this need `eslint-enable` followup? https://eslint.org/docs/2.13.1/user-guide/configuring#disabling-rules-with-inline-comments

Reply:
Oof, you're right. I thought it worked like //@ts-ignore